extensions/all-extensions/pom.xml

```diff
@@ -17,12 +17,17 @@
     <dependencies>
         <dependency>
             <groupId>org.mule.modules</groupId>
+            <artifactId>mule-module-ftp</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.mule.modules</groupId>
             <artifactId>mule-module-file</artifactId>
             <version>${project.version}</version>
         </dependency>
         <dependency>
             <groupId>org.mule.modules</groupId>
-            <artifactId>mule-module-ftp</artifactId>
+            <artifactId>mule-module-sockets</artifactId>
```

Comment:
this is ok for your local branch to function, but this cannot be merged. A rebase is needed instead